test/deploy/windows/kafka/install/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@NRhzhao same feedback, this should go in a /start step, instead of prepare

Reply:
@Julien4218 will update